articles/handling-bias-in-machine-learning/index.md

```diff
@@ -0,0 +1,151 @@
+
+[Image Source](https://images.unsplash.com/photo-1541844053589-346841d0b34c?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60)
+
+We are increasingly dependent on machine learning in our daily lives. An example is in the every-day products we use. However, these algorithms are created by human beings. As a result, algorithms may contain the same biases humans possess. An algorithm displaying say, sexist, or racist bias has the potential to cause more harm than good. It is thus important for algorithms to make predictions that are accurate and fair.
+
+In this article, we cover:
+
+1. What bias is in machine learning
+
+2. Types of bias.
+
+3. Why you should care about bias in machine learning algorithms.
+
+4. Techniques to handle bias.
+
+### Prerequisites
+
+Basic understanding of machine learning concepts. For an introduction to machine learning, check out this [article](https://www.section.io/engineering-education/supervised-learning-algorithms/). I also briefly covered the basics of machine learning in a previous [article](https://www.section.io/engineering-education/automated-fake-news-detection/).
+
+### Bias in Machine Learning
+
+[Wikipedia](https://en.wikipedia.org/wiki/Bias) describes bias as the disproportionate weight favoring or disfavouring an idea or thing in a closed-minded, unfair, or prejudicial way. Creators of machine learning models often unknowingly impart their various biases into their models. When this happens, it gives rise to what we refer to as algorithmic bias. Algorithmic bias refers to the methods in which algorithms portray bias of either their input data or their creators.
+
+In today’s world, machine learning models are used at such a large scale. They arguably drive our technology-dependent lives. Even so, if models happen to be biased, then biases are being mass-produced. It is an issue that may often end up being [ignored](https://www.technologyreview.com/2017/07/12/150510/biased-algorithms-are-everywhere-and-no-one-seems-to-care/) due to several reasons. One of the reasons could be that it is those underrepresented who are hit the hardest with the consequences of bias. We are also a bit too willing to trust the outcomes that these mathematical models make. It could also be that the creators of models and regulators may show little interest to monitor and inspect algorithms for inherent bias. They have little to no incentive to do so.
+
+It is especially sensitive when dealing with algorithms that involve race, sexual orientation, age, gender, and physical ability. Bias portrayed by these algorithms amplifies the societal issues that plague our current time, which artificial intelligence should be helping to reduce.
+
+We will look at why it is important to prevent bias but first, let’s look at a few types of bias.
+
+### Types of Bias
+
+Various types of bias occur in models and datasets that ultimately impact machine learning development. Let’s define a few.
+
+#### Exclusion Bias
```

Comment:
You've explained the types of bias really well.
It would help if you added one real world example each.

For example, for Prejudice Bias,
> Imagine a computer vision algorithm that is being trained to understand people at work. The algorithm is exposed to thousands of training data images, many of which show men writing code and women in the kitchen. The algorithm is likely to learn that coders are men and homemakers are women. This is prejudice bias because women obviously can code and men can cook.

Give such an example for each type of bias.

Reply:
@cyberShaw done! Check them out then let me know if all's good